python/README.md

```diff
@@ -10,32 +10,43 @@ For RefStudio, we bundle our Python application as a [resource](https://tauri.ap
 
 We use [poetry](https://python-poetry.org/) for Python dependency management. We also recommend using [pyenv](https://github.com/pyenv/pyenv) for Python version management.
 
+## Setting up Open AI environment variables
+
+```bash
+export OPENAI_API_KEY=your_key_here
+export OPENAI_CHAT_MODEL='gpt-3.5-turbo'
+```
+
+You can add these to your `.bashrc` or `.zshrc` to persist them.
+
```

Comment:
For development, you can also add a `.env` file inside the main refstudio directory (see `.env.example`).